.github/workflows/static-analysis.yml

```diff
@@ -24,8 +24,14 @@ jobs:
       - name: Install pnpm
         run: npm install -g pnpm
 
+      - name: Audit dependencies
+        run: pnpm audit --audit-level=low
+
       - name: Install dependencies
         run: pnpm install
 
       - name: Run Biome CI
         run: pnpm biome ci
+
+      - name: Run TypeScript type checking
+        run: pnpm run typecheck
```

Comment:
I accidentally broke the `eth` plugin while making changes to `handlers` files to enable `linea.eth` integration. This CI step will notify me if I do a similar mistake in the future.